src/main/java/com/petros/bibernate/session/PersistenceContext.java

```diff
@@ -0,0 +1,13 @@
+package com.petros.bibernate.session;
+
+public class PersistenceContext {
+
+    private static SessionFactory sessionFactory;
+
+    public static synchronized SessionFactory createSessionFactory() {
```

Comment:
Shouldn't SessionFactory create PersistenceContext instead? The SessionFactory is a higher-level abstraction and should be created during bootstrapping based on the given parameters. In my understanding, the SessionFactory creates the Session, and the PersistenceContext resides in the Session.

Reply:
It is another persistence context. Similar to Spring we need class that will create SessionFactory (singleton). I will rename it to Persistence